Accessing raw file content in github file kind (#2000)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/github/github.md

```
:::

> #### Accessing raw file content

Reviewer: ```suggestion
#### Ingest raw file content
```

> If you need to access the raw content of a file without parsing it, you can use the `skipParsing` key in your file selector. This is useful when you want to store the file content as a string or YAML property. When `skipParsing` is set to `true`, the file content will be kept in its original string format instead of being parsed into a JSON/YAML object.

Reviewer: ```suggestion
If you need to ingest the raw content of a file without parsing it, you can use the `skipParsing` key in your file selector.  
This is useful when you want to store the file content as a string or YAML property.  

When `skipParsing` is set to `true`, the file content will be kept in its original string format instead of being parsed into a JSON/YAML object.
```

> Here's an example that stores the raw content of a `values.yaml` file:

Reviewer: ```suggestion
Here's an example that ingests the raw content of a `values.yaml` file into the `content` property of a `file` entity:
```
